flixel/graphics/tile/FlxDrawTilesItem.hx

```diff
@@ -73,6 +82,13 @@ class FlxDrawTilesItem extends FlxDrawBaseItem<FlxDrawTilesItem>
 				tempFlags |= Tilesheet.TILE_RGB;
 			}
 			
+			#if (!openfl_legacy && openfl >= "3.5.4")
```

Comment:
Looks like OpenFL 3.6.0 just got released. These check would technically work, but should probably still be updated to avoid confusion about a non-existant 3.5.4 version.